doc/contributors/design-proposals/pull-bundle-on-a-cluster.md

```diff
@@ -0,0 +1,80 @@
+# Pull Bundle on a Cluster
+
+## Store a bundle in a `ConfigMap`
+Below is the directory layout of the operator bundle inside the image.
+```bash
+$ tree
+/
+├── manifests
+│   ├── testbackup.crd.yaml
+│   ├── testcluster.crd.yaml
+│   ├── testoperator.v0.1.0.clusterserviceversion.yaml
+│   └── testrestore.crd.yaml
+└── metadata
+    └── annotations.yaml
+    
+$ cat /annotations.yaml
+annotations:
+  operators.coreos.com.bundle.resources: "manifests+metadata"
+  operators.coreos.com.bundle.mediatype: "operator-registry+v1"
+```
+
+The following `ConfigMap` maps to the above operator bundle
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test
+  namespace: test
+  annotations:
+    operators.coreos.com.bundle.resources: "manifests+metadata"
+    operators.coreos.com.bundle.mediatype: "registry+v1"
+data:
+  testbackup.crd.yaml: content of testbackup.crd.yaml
+  testcluster.crd.yaml: content of testcluster.crd.yaml
+  testoperator.v0.1.0.clusterserviceversion.yaml: content oftestoperator.v0.1.0.clusterserviceversion.yaml
+  testrestore.crd.yaml: content of testrestore.crd.yaml
+```
```

Comment:
Is this generated by the user?

Reply:
This configmap is generated by some new code, which I'm going to describe at the beginning of this document in a bit (today).